src/ensembl/production/datacheck/static/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+
+    <title>Production Services</title>
+
+    <link href="http://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="/ensembl.css">
+    <link rel="shortcut icon" href="/img/production.png">
+
+  </head>
+
+  <body>
+
+    <div class="container-fluid">
+      <nav class="navbar navbar-default">
+        <div class="navbar-header">
+          <span class="navbar-brand">Production Services</span>
+        </div>
+        <ul class="nav navbar-nav">
+          <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button">Datachecks <span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><a class="dropdown-item" href="/datacheck/names">Datacheck Catalog</a></li>
+                <li><a class="dropdown-item" href="/datacheck/submit">Run Datachecks</a></li>
+              </ul>
+            </li>
+        </ul>
+      </nav>
+    </div>
+
+    <div class="container">
+      <h2><img src="/img/production.png" /> Production Services</h2>
+
+      <p>Roll up, roll up, we've got all the luverly <a href="/datacheck/names">datachecks</a> you'll ever need!</p>
```

Comment:
We probably don't need to keep this bit, where I was apparently channeling Derek Trotter...

Reply:
probably, yes. well spotted.